### ISSUE TITLE
Labels cause entire graph to not render in reading mode

### ISSUE THREAD

**Issue by invisi-splat** [Nigecat/obsidian-desmos#62]
**Description**
Labels cause entire graph to not render in reading mode.

**Expected behaviour**
When labels are applied to points, they should display along with the graph.

**Screenshots**
Code block:
<img width="732" alt="Screenshot 2022-05-12 at 11 21 56" src="https://user-images.githubusercontent.com/48893229/168049554-e51cb378-184a-4b35-98b9-0652943c1947.png">
Text version:
```desmos-graph
left=-7.5;right=7.5;
top=9;bottom=-1;
---
f(x)=3^x
g(x)=x+1|green
h(x)=4x-1.1|purple
(0,1)|green|label:(0,1), gradient ~ 1
```

Live preview (renders correctly):
<img width="633" alt="Screenshot 2022-05-12 at 11 23 44" src="https://user-images.githubusercontent.com/48893229/168049869-c3394ef3-e339-4990-9524-3e192f4dbe33.png">

Reading mode (renders incorrectly):
<img width="746" alt="Screenshot 2022-05-12 at 11 24 05" src="https://user-images.githubusercontent.com/48893229/168049932-22d06a7f-76a2-418d-97ff-5d95a2748c53.png">

**Desktop:**
 - OS: macOS Monterey 12.3.1
 - Obsidian Version: v0.14.6
 - Plugin Version: 0.6.3

**Additional context**
I am using Sanctum as a custom CSS theme, but the issue is still reproducible when using the default Obsidian theme.

Plotting points by themselves is fine. The issue only occurs when labelling points (and as soon as one point is labelled).

The graphs seem to stop rendering at the label content and instead display the label content in a code block.

I am unable to F12 and see the console logs as this does not work on Mac as far I as I can tell -- if there is a way, I would be more than happy to attach screenshots of the console logs.

**Comment by Nigecat** [comment]
I personally don't have a Mac device, but I think `Command+Option+I` might work to open the dev tools.  
 
Weirdly enough, it functions perfectly inside my dev vault but has problems when I try using that graph in my actual vault (so could potentially be a weird conflict with another plugin, not too sure). It seems as though I'm getting a slightly different issue though, as the graph is rendering just without the label being visible - though that's probably caused by the same problem - I'll have a look into it over the coming days.

**Comment by invisi-splat** [comment]
I've managed to open up the dev tools console, and it prints this whenever I render the graph:
`Invalid bounds argument. Expected a bounds object with bounds.bottom < bounds.top, and bounds.left < bounds.right. Got {"left":-7.5,"right":7.5,"bottom":null,"top":null}`

I've just opened up a new vault and loaded up the exact same plugins I have in my main vault - for some reason, the graph rendered fine. If there's a plugin conflict, it's a weird one.

I'm also sporadically having the same error you mentioned where the label doesn't render - in my vault, it also enlarges the graph hugely:
<img width="1175" alt="Screenshot 2022-05-12 at 13 23 58" src="https://user-images.githubusercontent.com/48893229/168073935-a7fa525c-1676-4733-85bd-3e24017d700e.png">

**Comment by Nigecat** [comment]
Yeah mine seems to be enlarging it for some reason too, would you happen to have the [Advanced Tables](https://github.com/tgrosinger/advanced-tables-obsidian) plugin installed?

After fiddling with my installed extensions, the problem seems to occur if I have the table plugin + this plugin installed, but disappears if the table plugin is disabled. Not too sure how it could be causing that though, but at least its a start.

**Comment by Nigecat** [comment]
I take that back, the problem is now occurring in my dev vault which has no other plugins in it. 
![image](https://user-images.githubusercontent.com/48661288/168454667-75df7a22-3583-4d4f-87c8-5eacc7859480.png)

**Comment by RIGIK93** [comment]
Try disabling caching in desmos-graph plugin settings. This is what usually causes all the trouble for me
